lxc/config.go

```diff
@@ -55,6 +55,9 @@ func (c *configCmd) usage() string {
 			"lxc config edit <container>                            Edit container configuration in external editor\n" +
 			"lxc config get <container> key                         Get configuration key\n" +
 			"lxc config set <container> key [value]                 Set container configuration key\n" +
```

Comment:
Not sure why we marked [value] optional for config set of the container but that seems wrong and should be changed, may as well do this in this pull request.